app-main/public/js/lessons/index.js

```diff
@@ -221,6 +221,13 @@ export function addLesson() {
     lessonInput.value = "";
     renderLessons(user);
   }
+      let submitLessonElement=document.getElementById("submit");
+       if(submitLessonElement.innerText=="UPDATE LESSON")
```

Comment:
I'd recommend sticking with the strict equality operator in JS (`===`). Otherwise you can end up with some strange type coercion bugs.

Reply:
Sure I will stick to (===) from now onwards and also I will change here and commit it.